### ISSUE TITLE
Support DUnitX framework

### ISSUE THREAD

**Issue by alefragnani** [alefragnani/delphi-generator-delphi#4]
Support **DunitX** as an alternative to **DUnit** when creating `Unit Test` projects

It should appear as the _last step_ when selecting `Unit Test` template, after deciding the _Runner Type_
```
└── Unit Test
    ├── Console
    │   ├── Text
    │   │   ├── DUnit
    │   │   └── DUnitX
    │   └── XML
    └── GUI
```

**Comment by alefragnani** [comment]
```
└── Unit Test
    ├── DUnit
    │   ├── Text
    │   └── XML
    └── DUnitX
```

**Comment by alefragnani** [comment]
Just released 1.2.0
